server/datastore/mysql/hosts.go

```diff
@@ -1909,3 +1910,34 @@ func (ds *Datastore) ListHostBatteries(ctx context.Context, hid uint) ([]*fleet.
 	}
 	return batteries, nil
 }
+
+// countHostNotResponding counts the hosts that haven't been submitting results for sent queries.
+//
+// Notes:
+//   - We use `2 * interval`, because of the artificial jitter added to the intervals in Fleet.
+//   - Default values for:
+//     - host.DistributedInterval is usually 10s.
+//     - svc.config.Osquery.DetailUpdateInterval is usually 1h.
+//   - Count only includes hosts seen during the last 7 days.
```

Comment:
Curious why count only hosts seen during last 7 days.

Reply:
Makes sense. Is 7 days the definition of MIA? (I thought it was 30 days, though I believe we are deprecating such thing in the UI/API)

Reply:
Yes, you're correct on the MIA definition and we are deprecating that. So I guess that category isn't really relevant and we're just using a seven day cutoff to correspond to the periodicity of the statistics job.